deploy/stacks/lambda_api.py

```diff
@@ -316,7 +316,7 @@ def create_function_role(self, envname, resource_prefix, fn_name, pivot_role_nam
                         'sts:AssumeRole',
                     ],
                     resources=[
-                        f'arn:aws:iam::*:role/{pivot_role_name}',
+                        f'arn:aws:iam::*:role/{pivot_role_name}-*',
```

Comment:
same as AssumeRle on `container.py` comment

Reply:
Fixed in https://github.com/data-dot-all/dataall/pull/1064/commits/66e65b0b54db09614cd23177e82f93249983db55 Great catch